CHANGES.rst

```diff
@@ -3,15 +3,14 @@
 Version 8.1.0
 -------------
 
-Unreleased
-
+-   An an argument to ``Path`` which mirrors writable/readable, but for
+    executable bits. :issue:`1961`
 
 Version 8.0.2
 -------------
 
 Unreleased
 
-
```

Comment:
Don't remove intentional formatting.

Reply:
I'm sloppy. I *think* I got it right this time. ;)